startup.js

```diff
@@ -0,0 +1,371 @@
+    network !== "emulator" ? `-f flow.${network}.json` : ""
+  }`;
+}
+
+function initializeKittyItems(network) {
+  if (!network) return envErr();
+  return `transactions send -o json --network=${network} --signer ${network}-account ./cadence/transactions/kittyItems/setup_account.cdc -f flow.json ${
+    network !== "emulator" ? `-f flow.${network}.json` : ""
+  }`;
+}
+
+//////////////////////////////////////////////////////////////////
+// ------------------- HELPER FUNCTIONS -----------------------
+//////////////////////////////////////////////////////////////////
+
+function convertToEnv(object) {
+  let envFile = "";
+  for (const key of Object.keys(object)) {
+    envFile += `${key}=${object[key]}\n`;
+  }
+  return envFile;
+}
+
+function envErr() {
+  throw new Error(
+    `Unknown or missing CHAIN_ENV environment variable.
+         Please provide one of the following: "emulator", "testnet"`
+  );
+}
+
+function adminError() {
+  throw new Error(
+    `Unknown or missing ADMIN_ADRESS environment variable.
+      Please create a testnet account and add your credentials to .env.tenstnet.local`
+  );
+}
+
+function deploy(chainEnv) {
+  switch (chainEnv) {
+    case "emulator":
+      return EMULATOR_DEPLOYMENT;
+    case "testnet":
+      return TESTNET_DEPLOYMENT;
+    default:
+      envErr();
+  }
+}
+
+async function generateKeys() {
+  const { stdout: out, stderr: err } = await exec(
+    `flow keys generate -o json`,
+    { cwd: process.cwd() }
+  );
+
+  if (err) {
+    console.log(err);
+  }
+
+  return JSON.parse(out);
+}
+
+function requireEnv(chainEnv) {
+  switch (chainEnv) {
+    case "emulator":
+      return ".env.local";
+    case "testnet":
+      if (process.env.APP_ENV === "local") return ".env.testnet.local";
+      throw new Error(
+        "Testnet deployment config not created. See README.md for instructions."
+      );
+    default:
+      envErr();
+  }
+}
+
+function runProcess(config, cb = () => {}) {
+  return new Promise((resolve, reject) => {
+    pm2.start(config, function (err, result) {
+      if (err) {
+        console.log(err);
+        reject(err);
+      }
+      resolve(result);
+    });
+  });
+}
+
+//////////////////////////////////////////////////////////////////
+// ------------- PROCESS MANAGEMENT ENTRYPOINT -------------------
+//////////////////////////////////////////////////////////////////
+
+pm2.connect(true, async function (err) {
+  if (err) {
+    console.error(err);
+    process.exit(2);
+  }
+
+  const spinner = ora();
+  spinner.spinner = "dots3";
+  spinner.color = "green";
+  let env = {};
+
+  // ------------------------------------------------------------
+  // ------------- EMULATOR DEPLOYMENT --------------------------
+
+  if (process.env.CHAIN_ENV === "emulator") {
+    spinner.start("Emulating Flow Network");
+
+    await runProcess({
+      name: "emulator",
+      script: "/Users/mackenziekieran/Downloads/flow-x86_64-darwin-",
```

Comment:
```suggestion
      script: "/Users/mackenziekieran/Downloads/flow-x86_64-darwin-",
```

I'm guess this was to test the dev wallet build? Just a reminder to remove before merge

Reply:
Yes we can remove this morning with the cli release.